FullStackMVCAPP/UnitTests/UnitTest1.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace UnitTests
+{
+    [TestClass]
+    public class UnitTest1
+    {
+        [TestMethod]
+        public void TestMethod1()
+        {
+        }
+    }
+}
```

Comment:
Remove if not neccessary

Reply:
hmm thats weird, I cant see a unit test called that in my local solution :/